.gitignore

```diff
@@ -20,14 +17,6 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-# Generated css files
-# src/**/*.css
-
-# App Binaries
-electron-build/
-
-src/images/profile_pic_2.jpeg
-
-out/
+app/
 dist/
 .idea/
```

Comment:
Could you re-add `build/` and `out/`? These are created after running `npm run pack`